src/app/pages/all-volunteers/all-volunteers.component.html

```diff
@@ -30,6 +31,12 @@
                   <td>
                     <a>{{volunteer.email}}</a>
                   </td>
+                  <td>
```

Comment:
i don't think you need this cell (<td>) for {{action}} what is the purspose anyway?